staging/src/k8s.io/api/storage/v1/types.go

```diff
@@ -221,6 +221,133 @@ type VolumeError struct {
 // +genclient:nonNamespaced
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
+// CSIDriver captures information about a Container Storage Interface (CSI)
+// volume driver deployed on the cluster.
+// CSI drivers do not need to create the CSIDriver object directly. Instead they may use the
+// cluster-driver-registrar sidecar container. When deployed with a CSI driver it automatically
```

Comment:
We do not support cluster-driver-registrar any longer: https://kubernetes-csi.github.io/docs/cluster-driver-registrar.html